__test__/user.test.js

```diff
@@ -0,0 +1,35 @@
+import app from "../app.js";
+import supertest from "supertest";
+import mongoose from "mongoose";
+import { config } from "dotenv";
+
+config();
+
+describe("users", () => {
+  beforeAll(async () => {
+    await mongoose.disconnect();
+    await mongoose.connect(process.env.MONGO_URI);
```

Comment:
Do not use production DB for unit tests. Please create some test DB for it

Reply:
ok I will change it, just a second